ckanext/etsin/mappers/iso_19139.py

```diff
@@ -145,10 +162,13 @@ def iso_19139_mapper(context, data_dict):
         pass
 
     # Date of formal issuance for the dataset
+    # TODO: This needs to be verified which field to use in iso19139 metadata:
+    # date-released, metadata-date, dataset-reference-date[0]['value'], date-created, date-updated
     try:
         package_dict['issued'] = iso_values['date-released']
     except KeyError:
-        pass
+        if 'dataset-reference-date' in iso_values and len(iso_values['dataset-reference-date']):
```

Comment:
Could more compact: if iso_values.get('key') be used?

Reply:
Does iso_values.get('key') throw an error if the 'key' cannot be found? Then we would need to have another try-except there?

Does iso_values.get('key'), in case the 'key' exists in the dict, return True only if the value for the key has a non-zero length? Or would it return True even if the value of 'key' was '' (empty)?